src/commands/project/deploy/start.ts

```diff
@@ -263,7 +263,7 @@ export default class DeployMetadata extends SfCommand<DeployResultJson> {
   }
 
   protected catch(error: Error | SfError): Promise<SfCommand.Error> {
-    if (error instanceof SourceConflictError) {
+    if (error instanceof SourceConflictError && error.data) {
```

Comment:
❓ : should this check the length of `error.data`?

It's a required arg:
https://github.com/forcedotcom/source-tracking/pull/511/files#diff-76b31628824755a528610210e965f349b1c66c2a2ac1e880cf02b4d3b1020a17R76

but STL could do `throw new SourceConflictError(`${conflicts.length} conflicts detected`, [])` and make this pass.